### PR TITLE
Remove references to defunct presto-accumulo tools repository

### PR DESCRIPTION
**Pull request by jhlodin**
## Description

~~The Accumulo connector uses a custom plugin that's currently all named around Presto. This PR proposes that we work with the plugin owners to shift from the "Presto" name into "Trino", and outlines what this doc should look like after such a change.~~

The Bloomberg presto-accumulo repository is not maintained anymore, so the conclusion is to no longer link to it.

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Documentation

> How would you describe this change to a non-technical end user or system administrator?

Update the Accumulo connector doc to be written for Trino.

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by mosabua** [comment]
@adamjshook do you think you could get the repo renamed to use trino in the name or updated? Or maybe clarify if the tools are even still needed. Maybe you could reach out to others at Bloomberg who might have access.

**Review by hashhar — DISMISSED** [review]

**Comment by hashhar** [comment]
ping @jhlodin - is this good to merge?

**Comment by jhlodin** [comment]
> ping @jhlodin - is this good to merge?

@hashhar As Yuya said, this PR is dependent on Bloomberg making changes to their plugin so it can't be merged until that's done. The goal with this PR is to provide guidance on the changes we would like to see there for the Trino rebrand.

**Comment by adamjshook** [comment]
I don't think Bloomberg's `presto-accumulo` repository are maintained anymore.  The main thing would be the iterators module, but that was strictly for use on JDK 8 which has long since been EOL. The other modules over there are just examples and some tooling which isn't necessary for the operation of the connector. If we'd like to move this functionality over to the trinodb org and update it, I can help with that.

Either way, in the short term, I think the notes and various links in this README to Bloomberg's `presto-accumulo` can be removed and I can add a note in Bloomberg's presto-accumulo repository that it is no longer maintained.

**Comment by adamjshook** [comment]
As I refresh my memory on the tools, the only remaining tool is this index migration tool which is used to migrate data to a different indexing strategy which was written years ago -- code which was never merged with Presto (or Trino). So there is really no need for this repository to even be moved to the trinodb org. I'm in favor of removing any links in this README; none of it is relevant.

**Comment by mosabua** [comment]
Excellent @adamjshook. It would be great if you could update the Bloomberg repo accordingly. 

@jhlodin can update this PR to remove the links and any references to the resources there. We will rely on your input and reviews @adamjshook and @ebyhr to get this technically accurate.

**Comment by adamjshook** [comment]
> It would be great if you could update the Bloomberg repo accordingly.

I've updated the Bloomberg repo.

File: docs/src/main/sphinx/connector/accumulo.rst [hunk 1]
```diff
@@ -59,7 +59,7 @@ Property name                                    Default value          Required
 ``accumulo.zookeepers``                          (none)                 Yes        ZooKeeper connect string
 ``accumulo.username``                            (none)                 Yes        Accumulo user for Trino
 ``accumulo.password``                            (none)                 Yes        Accumulo password for user
-``accumulo.zookeeper.metadata.root``             ``/presto-accumulo``   No         Root znode for storing metadata. Only relevant if using default Metadata Manager
+``accumulo.zookeeper.metadata.root``             ``/trino-accumulo``    No         Root znode for storing metadata. Only relevant if using default Metadata Manager
```
[the remark this answers is not in the document]
Comment (jhlodin) [reply docs/src/main/sphinx/connector/accumulo.rst:62]: Should that be changed?

File: docs/src/main/sphinx/connector/accumulo.rst [hunk 2]
```diff
@@ -617,7 +617,7 @@ follows:
     /metadata-root/schema/table
 
 Where ``metadata-root`` is the value of ``zookeeper.metadata.root`` in
-the config file (default is ``/presto-accumulo``), ``schema`` is the
+the config file (default is ``/trino-accumulo``), ``schema`` is the
```
[the remark this answers is not in the document]
Comment (jhlodin) [reply docs/src/main/sphinx/connector/accumulo.rst:620]: Should this be changed?

File: docs/src/main/sphinx/connector/accumulo.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhlodin) [reply docs/src/main/sphinx/connector/accumulo.rst]: Change to the default value here: https://github.com/trinodb/trino/pull/14326

File: docs/src/main/sphinx/connector/accumulo.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (jhlodin) [reply docs/src/main/sphinx/connector/accumulo.rst]: Change to the default value here: https://github.com/trinodb/trino/pull/14326

**Review by mosabua — APPROVED** [review]

**Review by ebyhr — APPROVED** [review]

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**